zerver/tests/test_events.py

```diff
@@ -2751,7 +2765,7 @@ def test_queries(self) -> None:
                     client_gravatar=False,
                 )
 
-        self.assert_length(queries, 29)
+        self.assert_length(queries, 47)
```

Comment:
This number going up by 18 is a sign that you're probably accidentally doing database queries in a loop.  Ideally, this would go up by 1-2 at most.